update-release-xcframeworks-static.sh

```diff
@@ -3,29 +3,35 @@ PROJECT=$PWD/MUXSDKBitmovin/MUXSDKBitmovin.xcworkspace
 TARGET_DIR=$PWD/XCFramework
 
 
-# Delete the old stuff                                                                                                                                                                         
+# Delete the old stuff
 rm -Rf $TARGET_DIR
 
 # Make the build directory                                                                                                                                                                     
 mkdir -p $BUILD_DIR
 # Make the target directory                                                                                                                                                                    
 mkdir -p $TARGET_DIR
 
-################ Build MuxCore SDK                                                                                                                                                             
+# Clean up on error
+clean_up_error () {
+    rm -Rf $BUILD_DIR
+    exit 1
+}
+
+################ Build MuxBitmovin SDK
 
 xcodebuild archive -scheme MUXSDKBitmovinTv -workspace $PROJECT -destination "generic/platform=tvOS" -archivePath "$BUILD_DIR/MUXSDKBitmovinTv.tvOS.xcarchive" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DIS\
-TRIBUTION=YES CLANG_ENABLE_MODULES=NO MACH_O_TYPE=staticlib
- xcodebuild archive -scheme MUXSDKBitmovinTv -workspace $PROJECT -destination "generic/platform=tvOS Simulator" -archivePath "$BUILD_DIR/MUXSDKBitmovinTv.tvOS-simulator.xcarchive" SKIP_INSTALL=NO \
-BUILD_LIBRARY_FOR_DISTRIBUTION=YES CLANG_ENABLE_MODULES=NO MACH_O_TYPE=staticlib
- xcodebuild archive -scheme MUXSDKBitmovin -workspace $PROJECT  -destination "generic/platform=iOS" -archivePath "$BUILD_DIR/MUXSDKBitmovin.iOS.xcarchive" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIB\
-UTION=YES CLANG_ENABLE_MODULES=NO MACH_O_TYPE=staticlib
- xcodebuild archive -scheme MUXSDKBitmovin -workspace $PROJECT  -destination "generic/platform=iOS Simulator" -archivePath "$BUILD_DIR/MUXSDKBitmovin.iOS-simulator.xcarchive" SKIP_INSTALL=NO BUILD\
-_LIBRARY_FOR_DISTRIBUTION=YES CLANG_ENABLE_MODULES=NO MACH_O_TYPE=staticlib
+TRIBUTION=YES CLANG_ENABLE_MODULES=NO MACH_O_TYPE=staticlib || clean_up_error
+xcodebuild archive -scheme MUXSDKBitmovinTv -workspace $PROJECT -destination "generic/platform=tvOS Simulator" -archivePath "$BUILD_DIR/MUXSDKBitmovinTv.tvOS-simulator.xcarchive" SKIP_INSTALL=NO \
```

Comment:
is it possible to define a function that takes the arguments to `xcodebuild` and calls `xcodebuild archive ... || clean_up_error`?

That way we don't duplicate `|| clean_up_error` on every line?

Reply:
It would have to take several arguments: the scheme, the destination, the build directory. But maybe haha I dont have too much experience with scripts, let me try it.